po/git-novice.ja.po

```diff
@@ -6311,38 +6306,32 @@ msgid ""
 "> > that are different."
 msgstr ""
 ">\n"
-"> The Owner pushed commits to the repository without giving any information\n"
-"> to the Collaborator. How can the Collaborator find out what has changed "
-"with\n"
-"> command line? And on GitHub?\n"
+"> オーナーはコラボレーターに情報を提供せずに、"
+"> コミットをリポジトリにプッシュしました。"
+"> どのようにしてコラボレーターは何が変更されたかをコマンドラインで、"
+"> またはGitHubで見つけることができるでしょうか?\n"
 ">\n"
-"> > ## Solution\n"
-"> > On the command line, the Collaborator can use ```git fetch origin "
-"master```\n"
-"> > to get the remote changes into the local repository, but without "
-"merging\n"
-"> > them. Then by running ```git diff master origin/master``` the "
-"Collaborator\n"
-"> > will see the changes output in the terminal.\n"
-"> >\n"
-"> > On GitHub, the Collaborator can go to their own fork of the repository "
-"and\n"
-"> > look right above the light blue latest commit bar for a gray bar saying\n"
-"> > \"This branch is 1 commit behind Our-Repository:master.\" On the far "
-"right of\n"
-"> > that gray bar is a Compare icon and link. On the Compare page the\n"
-"> > Collaborator should change the base fork to their own repository, then "
-"click\n"
-"> > the link in the paragraph above to \"compare across forks\", and "
-"finally\n"
-"> > change the head fork to the main repository. This will show all the "
-"commits\n"
-"> > that are different."
+"> > ## 解答\n"
+"> > コマンドラインで、コラボレーターは```git fetch origin master```\n"
+"> > を使用してマージすることなくリモートの変更をローカルリポジトリに"
+"> > 取得できます。次に ```git diff master origin/master``` を実行すると、"
+"> > コラボレーターはターミナルに出力された変更を確認できます。\n"
+"> >\n"
+"> > GitHubで、コラボレーターはリポジトリのフォークに移動し、"
+"> > “commits”をクリックして、"
```

Comment:
I don't see this in the original English version.

Reply:
I corrected the mistranslation at https://github.com/swcarpentry-ja/i18n/pull/147/commits/8d7bb459e7d463ec5ccd0c4859cea679a1759a26

Reply:
Looks good to me. BTW [in the most recent version](http://swcarpentry.github.io/git-novice/08-collab/index.html#solution) they seem to have removed this part. It was probably too confusing!